packages/mui-base/src/useMenu/MenuProvider.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
```

Comment:
The `...Provider` components are merely shortcuts that set up two contexts - CompoundComponentContext and ListContext. They are created for convenience.